src/HIDTables.h

```diff
@@ -48,655 +39,762 @@ THE SOFTWARE.
-#define HID_CONSUMER_AC_YES	0x25D	// HID type SEL
-#define HID_CONSUMER_AC_NO	0x25E	// HID type SEL
-#define HID_CONSUMER_AC_CANCEL	0x25F	// HID type SEL
-#define HID_CONSUMER_AC_CATALOG	0x260	// HID type SEL
-#define HID_CONSUMER_AC_BUY_SLASH_CHECKOUT	0x261	// HID type SEL
-#define HID_CONSUMER_AC_ADD_TO_CART	0x262	// HID type SEL
-#define HID_CONSUMER_AC_EXPAND	0x263	// HID type SEL
-#define HID_CONSUMER_AC_EXPAND_ALL	0x264	// HID type SEL
-#define HID_CONSUMER_AC_COLLAPSE	0x265	// HID type SEL
-#define HID_CONSUMER_AC_COLLAPSE_ALL	0x266	// HID type SEL
-#define HID_CONSUMER_AC_PRINT_PREVIEW	0x267	// HID type SEL
-#define HID_CONSUMER_AC_PASTE_SPECIAL	0x268	// HID type SEL
-#define HID_CONSUMER_AC_INSERT_MODE	0x269	// HID type SEL
-#define HID_CONSUMER_AC_DELETE	0x26A	// HID type SEL
-#define HID_CONSUMER_AC_LOCK	0x26B	// HID type SEL
-#define HID_CONSUMER_AC_UNLOCK	0x26C	// HID type SEL
-#define HID_CONSUMER_AC_PROTECT	0x26D	// HID type SEL
-#define HID_CONSUMER_AC_UNPROTECT	0x26E	// HID type SEL
-#define HID_CONSUMER_AC_ATTACH_COMMENT	0x26F	// HID type SEL
-#define HID_CONSUMER_AC_DELETE_COMMENT	0x270	// HID type SEL
-#define HID_CONSUMER_AC_VIEW_COMMENT	0x271	// HID type SEL
-#define HID_CONSUMER_AC_SELECT_WORD	0x272	// HID type SEL
-#define HID_CONSUMER_AC_SELECT_SENTENCE	0x273	// HID type SEL
-#define HID_CONSUMER_AC_SELECT_PARAGRAPH	0x274	// HID type SEL
-#define HID_CONSUMER_AC_SELECT_COLUMN	0x275	// HID type SEL
-#define HID_CONSUMER_AC_SELECT_ROW	0x276	// HID type SEL
-#define HID_CONSUMER_AC_SELECT_TABLE	0x277	// HID type SEL
-#define HID_CONSUMER_AC_SELECT_OBJECT	0x278	// HID type SEL
-#define HID_CONSUMER_AC_REDO_SLASH_REPEAT	0x279	// HID type SEL
-#define HID_CONSUMER_AC_SORT	0x27A	// HID type SEL
-#define HID_CONSUMER_AC_SORT_ASCENDING	0x27B	// HID type SEL
-#define HID_CONSUMER_AC_SORT_DESCENDING	0x27C	// HID type SEL
-#define HID_CONSUMER_AC_FILTER	0x27D	// HID type SEL
-#define HID_CONSUMER_AC_SET_CLOCK	0x27E	// HID type SEL
-#define HID_CONSUMER_AC_VIEW_CLOCK	0x27F	// HID type SEL
-#define HID_CONSUMER_AC_SELECT_TIME_ZONE	0x280	// HID type SEL
-#define HID_CONSUMER_AC_EDIT_TIME_ZONES	0x281	// HID type SEL
-#define HID_CONSUMER_AC_SET_ALARM	0x282	// HID type SEL
-#define HID_CONSUMER_AC_CLEAR_ALARM	0x283	// HID type SEL
-#define HID_CONSUMER_AC_SNOOZE_ALARM	0x284	// HID type SEL
-#define HID_CONSUMER_AC_RESET_ALARM	0x285	// HID type SEL
-#define HID_CONSUMER_AC_SYNCHRONIZE	0x286	// HID type SEL
-#define HID_CONSUMER_AC_SEND_SLASH_RECEIVE	0x287	// HID type SEL
-#define HID_CONSUMER_AC_SEND_TO	0x288	// HID type SEL
-#define HID_CONSUMER_AC_REPLY	0x289	// HID type SEL
-#define HID_CONSUMER_AC_REPLY_ALL	0x28A	// HID type SEL
-#define HID_CONSUMER_AC_FORWARD_MSG	0x28B	// HID type SEL
-#define HID_CONSUMER_AC_SEND	0x28C	// HID type SEL
-#define HID_CONSUMER_AC_ATTACH_FILE	0x28D	// HID type SEL
-#define HID_CONSUMER_AC_UPLOAD	0x28E	// HID type SEL
-#define HID_CONSUMER_AC_DOWNLOAD_(SAVE_TARGET_AS)	0x28F	// HID type SEL
```

Comment:
Incidentally, this PR also fixes an old bug where this keycode macro was misnamed. This was fixed in Kaleidoscope quite a while ago.